source/_integrations/lg_thinq.markdown

```diff
@@ -98,6 +100,67 @@ A read-only property which has only two states that can be toggled is represente
+| [Home Brew](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Home-Brew/)| Recipe name<br>Wort info<br>Yeast info<br>Hop oil info<br>Flavor info<br>Beer remain |
+| | Current state |
+| | Elapsed day on current status<br>Total elapsed day |
+| [Hood](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Hood/) | Hood operation mode |
+| | Remained time |
+| Humidifier | PM1<br>PM2<br>PM10<br>Humidity<br>Total pollution<br>Enable monitoring<br>Temperature |
+| | Operation timer |
+| | Sleep timer |
+| [Kimchi Refrigerator](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Kimchi-Refrigerator/) | Fresh air filter mode<br>Enable one touch filter |
+| | Target temperature |
+| [Microwave Oven](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Microwave-Oven/) | Current state |
+| | Remained time |
+| [Oven](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Oven/) | Current state |
+| | Target temperature |
+| | Remained time |
+| [Plant Cultivator](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Plant-Cultivator/) | Brightness<br>Duration<br>Start time<br>End time |
+| | Current state<br>Growth mode<br>Wind volume |
+| | Daytime target temperature<br>Night target temperaturet<br>Temperature |
+| [Refrigerator](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/refrigerator/) | Door state |
+| | Fresh air filter mode |
+| | Used time |
+| [Robot Cleaner](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/robot-cleaner/) | Current state |
+| | Current job mode |
+| | Running time |
+| [Stick Cleaner](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Stick-Cleaner/) | Current state |
+| | Current job mode |
+| | Battery level |
+| [Styler](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/styler/) | Current state |
+| | Remained time<br>Operation timer<br>Total time |
+| Washcombo Main<br>Washcombo Mini<br>[Washer](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/washer/)<br>[Washtower Dryer](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/WashTower-Dryer/)<br>[Washtower](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/WashTower-Single-Unit/)<br>[Washtower](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/WashTower-Single-Unit/)<br>[Washtower Washer](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/WashTower-Washer/) | Current state |
+| | Operation timer |
+| [Water Heater](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/Water-Heater/) | Operation mode |
+| | Current temperature |
+| [Water Purifier](https://thinq.developer.lge.com/en/cloud/docs/thinq-connect/device-profile/water-purifier/) | Cock state<br>Sterilizing state |
```

Comment:
what is cock state? I see in the code it's called UVnano. But what is it? The idea is to list the functionality as the user sees it in the frontend, not to provide developer information.

Reply:
Update.
Cock is body.

Reply:
I am sorry, I still don't quite understand.
This is a water purifier. What does a body do in there?
Also: "cock" is also a [vulgar term for penis](https://www.merriam-webster.com/dictionary/cock), see meaning 5. This is the absolute first thing I thought of when I read this :blush:  
Do you think it would be possible to remove the term "cock" entirely? Or is there another word we could use instead?

Reply:
> I am sorry, I still don't quite understand. This is a water purifier. What does a body do in there? Also: "cock" is also a [vulgar term for penis](https://www.merriam-webster.com/dictionary/cock), see meaning 5. This is the absolute first thing I thought of when I read this 😊 Do you think it would be possible to remove the term "cock" entirely? Or is there another word we could use instead? does it mean "Cook", maybe? As in heat the water to boiling point?

I am sorry that my explanation was not enough.
Cock state can have normal and cleaning.
In normal mode, you get clean and cold water.
In cleaning mode, you get clean water.

Reply:
How can I express easy way for a user? Water mode?

Reply:
I'm not entirely sure if I understand it correctly. Maybe something like:

Purification mode : Normal / Cleaning
- Normal: Normal operation, not sterilizing
- Cleaning: In sterilizing mode.

Reply:
I updated with 'Purification mode'.
But I removed options 'Normal / Cleaning', for consistency with other property names.